File: src/main/java/com/snackgame/server/applegame/exception/InaccuratePercentileException.java
package com.snackgame.server.applegame.exception;

import com.snackgame.server.common.exception.Kind;

public class InaccuratePercentileException extends AppleGameException {

    public InaccuratePercentileException(double percentile) {
        super(Kind.INTERNAL_SERVER_ERROR, "백분위 계산이 잘못되었습니다: " + percentile);
    }
}

File: src/main/java/com/snackgame/server/game/snackgame/exception/InaccuratePercentileException.kt
package com.snackgame.server.game.snackgame.exception

import com.snackgame.server.common.exception.Kind

class InaccuratePercentileException(percentile: Double) : SnackgameException(
    "백분위 계산이 잘못되었습니다: $percentile", Kind.INTERNAL_SERVER_ERROR
)
